react-native/react/native/remote-store.desktop.js

```diff
@@ -5,17 +5,13 @@ export default class RemoteStore {
   listeners: Array<Function>;
   internalState: any;
 
-  constructor (props: {}) {
+  constructor (props: Object) {
```

Comment:
ah I see you're passing two new props, you can specify that instead of Object which will rarely catch anything